index.html

```diff
@@ -15,6 +15,7 @@
 		:root {
 			--parley-mobile-height: 1vh;
 			--parley-mobile-width: 1vw;
+			--parley-index-prefix: 100;
```

Comment:
Kan je de var name veranderen naar `--parley-base-z-index`?
Net wat duidelijker dan `index-prefix`

Reply:
En zet die var name ook ff in de PR in je introductie tekst.
```
CSS var name that can be changed to influence the base z-index is `--parley-base-z-index`
```

Reply:
Done

Reply:
Resolved, var naam is aangepast